src/handler.ts

```diff
@@ -24,13 +24,19 @@ export async function handlePullRequest (context: Context): Promise<void> {
 
   const owner = payload.repository.owner.login
   const title = payload.pull_request.title
+  const sender = payload.sender.login
 
   if (config.skipKeywords && includesSkipKeywords(title, config.skipKeywords)) {
     context.log('skips adding reviewers')
     return
   }
 
-  const reviewers = chooseUsers(owner, config.reviewers, config.numberOfReviewers)
+  const reviewers = chooseUsers(
+    owner,
```

Comment:
This can use `sender` instead of `owner`. I think it is my mistake that `owner` points to the repository owner (I want to do is to not include the PR author).

```typescript
const reviewers = chooseUsers(
    sender,
    config.reviewers,
    config.numberOfReviewers
  )
```

Reply:
I totally agree - I'll adjust PR accordingly